frontend/src/general/VerticalTabs.jsx

```diff
@@ -33,6 +33,7 @@ export default class VerticalTabs extends React.Component {
           aria-label="Vertical tabs"
           className={`${tabs} lightGrey blackFont flexContents`}
           orientation="vertical"
+          indicatorColor="primary"
```

Comment:
I think that color shows up as purple on my browser. Is that our primary color?
![image](https://user-images.githubusercontent.com/35940434/106308743-26ddff00-622f-11eb-9a9f-eba36af9b45d.png)

Reply:
Yeah that color is set to our primary, I think it's the default one by Material UI.